### DISCUSSION TITLE
Mi band 4 config

### DISCUSSION THREAD

**Discussion by Slejpnir** [mKeRix/room-assistant#630] [Q&A] [answered]
Hello,
Can somebody share your config file for Pi Zero W with Mi band 4 detection?

**Comment by spkr4thecloud** [comment, marked as answer]
just an FYI the following is for a MiBand5:

-----------------------------------------------------------

global:
  instanceName: livingroom
  integrations:
    - homeAssistant
    - bluetoothLowEnergy
homeAssistant:
  mqttUrl: 'mqtt://[ipaddres of ha]:1883'
  mqttOptions:
    username: [your_mqtt_username]
    password: [your_mqtt_password]
cluster:
    autoDiscovery: false
    networkInterface: wlan0
    port: 6425
    peerAddresses:
      - 192.168.0.120:6425
      - 192.168.0.121:6425
      - 192.168.0.122:6425
      - 192.168.0.123:6425
      - 192.168.0.28:6425
bluetoothLowEnergy:
  whitelist:
    - [mibandmacaddress]
  maxDistance: 9
---------------------------------------------------------------
you put the mac address of your mi band where i have put [mibandmacaddress]

an example would be:

bluetoothLowEnergy:
  whitelist:
    - ab12cd34ef56

as always replace everything inside square brackets, including square bracket. 

hope this helps
